Makefile

```diff
@@ -229,7 +229,7 @@ update:
 shellcheck:
 	@set -e
 	@echo "checking parachain scripts..."
-	@find scripts -name "*.sh" | xargs shellcheck -S warning
+	@find scripts/ -name "*.sh" | xargs shellcheck -S warning
```

Comment:
why need the slash here? There are also shell scripts not only inside 'scripts' folder.

Reply:
So currently, When we create new scripts we put it under `scripts` folder and under the `scripts/litentry/` , Are we including scripts anywhere else?

Reply:
`make shellcheck` is a sality check for all shell script within the repo. Somewhere else inside the repo, there are shell scripts.